db/migrate/20160117215448_create_users.rb

```diff
@@ -1,6 +1,8 @@
 class CreateUsers < ActiveRecord::Migration
   def change
     create_table :users do |t|
+      t.string :name
```

Comment:
Just thinking; is there a way to aggregate all user information in one table? `email`, `name` are in `users` table while, `profile_picture`, `profile_url`, `profile_email` are in `social_providers` :thought_balloon: